app/descriptor/views.py

```diff
@@ -93,15 +93,15 @@ def edit_name(desc_id):
     old_name = descriptor.name
     form = EditDescriptorNameForm()
     if form.validate_on_submit():
-        descriptor.name = form.name.data
         if Descriptor.query.filter(Descriptor.name == form.name.data).first() \
```

Comment:
I'll show a warning that there is no change, and we just won't do the change -- so then we don't have to make an uneccessary db change?